init.sql

```diff
@@ -0,0 +1,10 @@
+GRANT ALL PRIVILEGES ON app_db.* TO 'app_user'@'%';
+
+-- Use the 'app' database
+USE app_db;
+
+-- Create a table to store counter data
+CREATE TABLE counter (
+   id INT AUTO_INCREMENT PRIMARY KEY,
+   value INT
+);
```

Comment:
The same, add EOF 
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/1980cef6-305f-4315-9965-e2583a2b0563)